### PR TITLE
ninas happy thoughts

### PR DESCRIPTION
**Pull request by ninamansster**
Hi, 
here is the links to my projects and its published sites:
https://ninamansster-happy-thoughts.netlify.com/
https://happy-thoughts-ninamansster.herokuapp.com/

Thanks,
Nina

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -7,23 +7,155 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
+//     default: false
+//   },
+//   createdAt: {
+//     type: Date,
+//     default: Date.now
+//   }
+// });
+
+// Message for Happy Thought, validation of 5-140 characters, hearts and created date.
+const Thought = mongoose.model('Thought', {
```
Comment (JennieDalgren) [inline server.js]: ⭐️

File: server.js [hunk 2]
```diff
@@ -7,23 +7,155 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
+//   },
+//   complete: {
+//     type: Boolean,
+//     default: false
+//   },
+//   createdAt: {
+//     type: Date,
+//     default: Date.now
+//   }
+// });
```
Comment (JennieDalgren) [inline server.js]: remove this

File: server.js [hunk 3]
```diff
@@ -7,23 +7,155 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
 
-// Add middlewares to enable cors and json body parsing
+//Middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+// ROUTES
```
Comment (JennieDalgren) [inline server.js]: Also add the error handling for the database connection. Check Damiens video about setting that middleware up here:
https://technigo.wistia.com/medias/l5wqrcu4to?wtime=9m22s

File: server.js (hunk 2 above)
Comment (ninamansster) [reply server.js]: Ooops, my bad, I removed stuff below, but didn't see this huge passage..🙀

File: server.js (hunk 3 above)
Comment (ninamansster) [reply server.js]: I don't get what you are asking for, Jennie? Can please clarify?

**Pull request closed without merging**